### ISSUE TITLE
Add examples for running code and requirements,txt

### ISSUE THREAD

**Issue by charmerDark** [sakethbachu/liveness_detection#1]
Please consider adding examples on how to run the program and a requirements.txt for reproducibility.

**Comment by EgoPingvina** [comment]
Yes that would be great. It is interesting example, but what should we do if don't have enough applied experience to run it on our own
